docs/tutorials/articles/3rd_party_components/index.md

```diff
@@ -93,9 +93,20 @@ Gui.register_content_provider(Map, expose_folium)
 
 Finally, we can embed the Folium Map using the following the `part` component:
 
-```
-<|part|content={folium_map()}|>
-```
+
+=== "Markdown"
+    ```
+    <|part|content={folium_map()}|>
+    ```
+=== "Python"
+    ```python
+    import taipy.gui.builder as tgb
+
+    with tgb.Page() as page:
+        ...
+        tgb.part(content="{folium_map()}")
+    ```
```

Comment:
Don't we want to put Python first? I thought that was what our users preferred.

Reply:
We didn't have this discussion. If we do and proceed with putting Python first, it should be for every code